internal/commands/build.go

```diff
@@ -37,6 +37,19 @@ type BuildFlags struct {
 	DefaultProcessType string
 }
 
+func validateBuildFlags(flags BuildFlags, logger logging.Logger, cfg config.Config, packClient PackClient) error {
+	if flags.Builder == "" {
+		suggestSettingBuilder(logger, packClient)
+		return MakeSoftError()
+	}
+
+	if flags.Registry != "" && !cfg.Experimental {
+		return errors.Errorf("Support for using buildpackages in a CNB Registry is currently experimental.")
```

Comment:
"buildpacks" may be clearer in this context

Reply:
I agree. I'm waiting for a respond on [this comment](https://github.com/buildpacks/pack/issues/608#issuecomment-627428147) and then I'll change the error message accordingly.